pretty/src/main/java/org/aya/pretty/backend/string/custom/UnixTermStyle.java

```diff
@@ -11,4 +11,6 @@
   Blink,
   Overline,
   Reverse,
+  TerminalRed,
+  TerminalGreen,
 }
```

Comment:
These are not like `Style.color(0xFF0000)` which set the absolute color, instead, they are decided by the terminal (like user-customized color scheme)